src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/add_command_request.cc

```diff
@@ -90,19 +90,6 @@ void AddCommandRequest::Run() {
     return;
   }
 
-  if ((*message_)[strings::msg_params].keyExists(strings::cmd_icon)) {
-    mobile_apis::Result::eType verification_result = MessageHelper::VerifyImage(
```

Comment:
I do not see MessageHelper::VerifyImage() is removed, do we still need it?

Reply:
Yes. It's reused in CreateInteractionChoiceSet.